src/sql/workbench/parts/dashboard/widgets/properties/media/propertiesWidget.css

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+body {
```

Comment:
>body [](start = 0, length = 4)

These are going to apply to all elements in ADS with these classes which I assume you don't want. Would suggest giving them more specific names (or embedding in a div with a specific id and then appending to these selector)

Reply:
👍 for any CSS we should scope by default. Likely under the properties widget selector